controllers/metallb_controller_test.go

```diff
@@ -16,12 +16,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const MetalLBTestNameSpace = "metallb-test-namespace"
```

Comment:
nit: I'd move this to `suite_test.go` , as it's shared between the two test files and the suite file is the first place where we use it.